frontend/src/lib/components/shared/Dropdown/Selectbox.tsx

```diff
@@ -182,13 +194,14 @@ class Selectbox extends React.PureComponent<Props, State> {
           )}
         </WidgetLabel>
         <UISelect
-          clearable={false}
           disabled={disabled}
           labelKey="label"
           aria-label={label || ""}
-          onChange={this.onChange}
+          onChange={this.onChange as (params: OnChangeParams) => unknown}
           onInputChange={this.onInputChange}
           onClose={this.onClose}
+          clearable={clearable}
```

Comment:
nit: we probably can also set `escapeClearsValue={clearable}` here as well

Reply:
Updated 👍